### PR TITLE
Gradient parser changes

### PR DESCRIPTION
**Pull request by mgierlasinski**
**Gradient Parser 2.0 aka Magic Crawler**
- create new WPF project
- merge UWP and Console into single app
- provide configuration describing all endpoints - generate all gradients, categories and metadata with one button click

### CODE REVIEW

File: Playground/Playground.Data/Resources/Popular.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
...
+    "tags": [
+      "popular"
+    ]
+  }
```
Comment (bbenetskyy) [inline Playground/Playground.Data/Resources/Popular.json:8]: Why do you add this one gradient?

File: Tools/MagicCrawler/MagicCrawler/Views/MainWindow.xaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        <controls:WebView x:Name="WebView" Grid.Row="1" Visibility="Collapsed" />
+        <DataGrid Grid.Row="1" ItemsSource="{Binding Jobs}"  AutoGenerateColumns="False">
+            <DataGrid.Columns>
+                <DataGridTextColumn Header="Title" Binding="{Binding Title}" IsReadOnly="True"/>
+                <DataGridHyperlinkColumn Header="Input" Binding="{Binding Input}" IsReadOnly="True"/>
+                <DataGridTextColumn Header="Output" Binding="{Binding Output}" IsReadOnly="True"/>
+                <DataGridTextColumn Header="Status" Binding="{Binding Status}" IsReadOnly="True" Width="80"/>
+            </DataGrid.Columns>
+        </DataGrid>
+        <!--<ListView Grid.Row="1" ItemsSource="{Binding Jobs}" HorizontalContentAlignment="Stretch">
```
Comment (bbenetskyy) [inline Tools/MagicCrawler/MagicCrawler/Views/MainWindow.xaml:37]: I think you can remove it 😉

File: Playground/Playground.Data/Playground.Data.csproj [hunk 3]
```diff
@@ -56,24 +59,28 @@
...
     <EmbeddedResource Include="Resources\StandardCircle.json" />
     <EmbeddedResource Include="Resources\Stripes.json" />
+    <EmbeddedResource Include="Resources\Texture.json" />
 	  <EmbeddedResource Include="Resources\Themes.json" />
-    <EmbeddedResource Include="Resources\SimplePatterns.json" />
   </ItemGroup>
 
   <ItemGroup>
     <PackageReference Include="LiteDB" Version="4.1.4" />
+    <PackageReference Include="Xamarin.Essentials" Version="1.5.3.2" />
```
Comment (bbenetskyy) [inline Playground/Playground.Data/Playground.Data.csproj:83]: Why did you add this NuGet?

File: Playground/Playground.Data/Resources/Popular.json (hunk 1 above)
Comment (mgierlasinski) [reply Playground/Playground.Data/Resources/Popular.json:8]: Gradients in each category are tested for finding duplicates in other categories. If there is something in Popular and for example Simple, gradient will not be added to Popular.json but tag popular will be added to Simple.json. In this case gradient in category Popular was not found anywhere else so it's added here.

File: Tools/GradientParser/GradientParser.ConsoleApp/Program.cs [hunk 4]
```diff
@@ -1,34 +1,58 @@
...
 {
     class Program
     {
-        static void Main(string[] args)
+        static async Task Main(string[] args)
         {
-            Console.WriteLine("Hello World! I'm Gradient Parser.");
-            Console.WriteLine();
-            Console.WriteLine("Please give me link to html file with gradients.");
+            //Console.WriteLine("Hello World! I'm Gradient Parser.");
```
[the remark this answers is not in the document]
Comment (mgierlasinski) [reply Tools/GradientParser/GradientParser.ConsoleApp/Program.cs]: Maybe I can revert it if you are sad. At first I just wanted to modify only console but in the end I created different app and probably forgot to revert those changes.

File: Tools/MagicCrawler/MagicCrawler/Views/MainWindow.xaml (hunk 2 above)
Comment (mgierlasinski) [reply Tools/MagicCrawler/MagicCrawler/Views/MainWindow.xaml:37]: That was first idea of list element but I switched to Grid, not sure if this is final :)

File: Playground/Playground.Data/Playground.Data.csproj (hunk 3 above)
Comment (mgierlasinski) [reply Playground/Playground.Data/Playground.Data.csproj:83]: Storing last update date in preferences.

File: Tools/GradientParser/GradientParser.ConsoleApp/Program.cs (hunk 4 above)
[the remark this answers is not in the document]
Comment (bbenetskyy) [reply Tools/GradientParser/GradientParser.ConsoleApp/Program.cs]: If we still plan to use this app - then we should clear all unneeded lines and it would be really nice if we will see in output some friendly greeting.

Otherwise, remove it all 😛

[the remark this answers is not in the document]
Comment (mgierlasinski) [reply Tools/GradientParser/GradientParser.ConsoleApp/Program.cs]: I double checked this and changes were reverted in later commits, you are looking at partial PR.
